prusti-interface/src/specs/mod.rs

```diff
@@ -54,170 +40,63 @@ pub struct SpecCollector<'tcx> {
     tcx: TyCtxt<'tcx>,
     extern_resolver: ExternSpecResolver<'tcx>,
 
-    /// Collected assertions before deserialisation.
-    spec_items: Vec<SpecItem>,
+    /// Map between specification ids and their corresponding function
```

Comment:
`corresponding function` here is inteded to mean "the Rust function that contains the typed body of the specification", right? At a first glance this might be misunderstood as `corresponding function` being the function to which the specs are attached. Maybe:
```suggestion
    /// Map between specification ids and their corresponding typed expressions.
```

Reply:
Addressed by 1bbbdfb4b539f253b77aef890ba056f8df9197e9